src/main/java/knightminer/inspirations/library/InspirationsTags.java

```diff
@@ -0,0 +1,49 @@
+package knightminer.inspirations.library;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.tags.ItemTags;
+import net.minecraft.tags.Tag;
+
+public class InspirationsTags {
+
+	public static class Blocks {
+		public static final Tag<Block> MULCH = new BlockTags.Wrapper(Util.getResource("mulch"));
+		public static final Tag<Block> SMALL_FLOWERS = new BlockTags.Wrapper(Util.getResource("small_flowers"));
+		public static final Tag<Block> FLOWER_POTS = new BlockTags.Wrapper(Util.getResource("flower_pots"));
+		public static final Tag<Block> CARPETED_TRAPDOORS = new BlockTags.Wrapper(Util.getResource("carpeted_trapdoors"));
+		public static final Tag<Block> CARPETED_PRESSURE_PLATES = new BlockTags.Wrapper(Util.getResource("carpeted_pressure_plates"));
+		public static final Tag<Block> BOOKSHELVES = new BlockTags.Wrapper(Util.getResource("bookshelves"));
+		public static final Tag<Block> ENLIGHTENED_BUSHES = new BlockTags.Wrapper(Util.getResource("enlightened_bushes"));
+
+		// Blocks with this tag act as fire for the cauldron.
```

Comment:
```suggestion
		/** Blocks with this tag act as fire for the cauldron. */
```

Should use inline javadoc for these, so it shows in the tooltip